File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.py
class Solution:
    def minBitFlips(self, start: int, goal: int) -> int:
        cnt = 0
        
        while start != 0 or goal !=0:
            print(start, goal)
            if start%2 != goal%2:
                cnt += 1
            start //= 2
            goal //= 2
        return cnt

File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.swift
class Solution {
    func minBitFlips(_ start: Int, _ goal: Int) -> Int {
        var res = 0
        var xor = start ^ goal

        while xor > 0 {
            res += xor & 1
            xor >>= 1
        }

        return res
    }
}